### PR TITLE
feat: see suggestions list

### PR DESCRIPTION
**Pull request by ManuViola77**
## Links:

[Agregar una pagina de sugerencias/feedback para que la gente pueda proponer mejoras y votarlas](https://github.com/users/ManuViola77/projects/1/views/1#:~:text=Agregar%20una%20pagina%20de%20sugerencias/feedback%20para%20que%20la%20gente%20pueda%20proponer%20mejoras%20y%20votarlas)

## What & Why:

This PR adds a Suggestions Page, where for now you can just see the suggestions and filter by status. In next PRs we will be able to add new suggestions and vote for the existing ones.

## Risks, Mitigation & Rollback:

- [ ] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

File: src/hooks/useSuggestions.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+          status,
+          title,
+          ...restSuggestionsProps
+        } = doc.data();
+        const negativeVotesCount = negativeVotes.length;
+        const positiveVotesCount = positiveVotes.length;
+        const statusColor =
+          SUGGESTIONS_STATUS.find(({ value }) => value === status)?.color || 'black';
+        const suggestedBy = positiveVotes[0];
+        const votedNegative = !!negativeVotes.find(({ id }) => id === myId);
```
Comment (alandours) [inline src/hooks/useSuggestions.js]: Here you could use `some()` that will already return a boolean if any element matches
```suggestion
        const votedNegative = negativeVotes.some(({ id }) => id === myId);
```

File: src/hooks/useSuggestions.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        const votedPositive = !!positiveVotes.find(({ id }) => id === myId);
```
Comment (alandours) [inline src/hooks/useSuggestions.js]: Same here

File: src/hooks/useSuggestions.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+import { SUGGESTIONS_STATUS } from 'constants/types';
+import { SUGGESTIONS } from 'firebase/collections';
+import firebaseData from 'firebase/firebase';
+import useAuth from 'hooks/useAuth';
+
+const { firebaseDb } = firebaseData;
+
+const useSuggestions = () => {
+  const DEFAULT_STATUS = useMemo(
+    () => SUGGESTIONS_STATUS.filter(({ isDefault }) => isDefault)[0],
```
Comment (alandours) [inline src/hooks/useSuggestions.js]: I think you could also use `find()` here and avoid needing to access the index
```suggestion
    () => SUGGESTIONS_STATUS.find(({ isDefault }) => isDefault),
```

File: src/components/SuggestionCard/index.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    statusColor,
+    suggestedBy: { name, photo },
+    title,
+    votedNegative,
+    votedPositive,
+  },
+}) => {
+  const t = useTranslation();
+
+  const votedColor = 'action';
```
Comment (alandours) [inline src/components/SuggestionCard/index.js]: Maybe this constant could be moved somewhere in `/constants`?

**Review by alandours — APPROVED** [review]
Looks good! I just left a few comments but they're not blockers

File: src/hooks/useSuggestions.js (hunk 1 above)
Comment (ManuViola77) [reply src/hooks/useSuggestions.js]: Awesome, thanks!!! 🙌🏻

File: src/hooks/useSuggestions.js (hunk 3 above)
Comment (ManuViola77) [reply src/hooks/useSuggestions.js]: great idea, thanks!

File: src/hooks/useSuggestions.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  const {
+    statusFilter: { value: statusFilterValue },
+  } = filters;
+
+  const {
+    user: { uid: myId },
+  } = useAuth();
+
+  useEffect(() => {
+    (async function () {
```
Comment (jeisux92) [inline src/hooks/useSuggestions.js]: i recommend you move this to a new method and then call it from here

Comment (jeisux92) [reply src/hooks/useSuggestions.js]: if im not wrong you already did it in a later PR

Comment (ManuViola77) [reply src/hooks/useSuggestions.js]: yes I was going to say that. I did that in a later PR 😄

**Pull request merged**